### DISCUSSION TITLE
GitHub Copilot

### DISCUSSION THREAD

**Discussion by hamzafarooq1408** [AammarTufail/python_ka_chilla2023#50] [Q&A]
I have complete access to GitHub Copilot Student Developer Pack but it has stopped working. I tried uninstalling and installing again and again but I keep getting the errors showing the following screenshot.

![Untitled](https://user-images.githubusercontent.com/116020865/211483250-5acc70c2-3845-43c6-8935-13c8657bfb1f.png)

**Comment by MuhammadHammad4127** [comment]
Hy! Hamza neechay bottom may option araha hy signup to Github Co Pilot usay click krien gay to app Github ki website pay directly chalein jayein gay... wahan aik option atta hy Allow ka usay allow kr k ap save krein to yahan VS code may hojaye ga... nahi to VS Code ko proper off kr k dubara open krein ... ihave also the same issue but is trah resolve hogiya tha.. Follow these steps then let us know it's working or not.

**Comment by SakibSumon** [comment]
hello @hamzafarooq1408, how are things now? copilot working?

**Comment by AzamBukhari** [comment]
> I think I need .edu email address. Which unfortunately I don't have. 
![image](https://user-images.githubusercontent.com/121688289/215550387-f8640941-c9c0-4548-98c4-480545b44b8f.png)

**Reply by Muhammad-Ali-Butt** [reply]
yes :( you can try tabnine as it has a free plan but it will defo not be as good as copilot but still worth a try

**Comment by SakibSumon** [comment]
you can use it for 60 days for free and if money is not an issue, I suggest you to give it a shot.
